src/wp-admin/includes/dashboard.php

```diff
@@ -1879,8 +1904,14 @@ function wp_dashboard_php_nag() {
 function dashboard_php_nag_class( $classes ) {
 	$response = wp_check_php_version();
```

Comment:
We use the `$check_php` variable in `dashboard_php_nag_class()`, but we use `$response` here; should we use `$check_php` here? 

Reply:
FYI: This could be updated given the change to another `$response` variable earlier in the PR. However, it should be noted that we're going through the files and updating such variables for readability anyway. I don't mind if the variable name is changed in this PR, or if it waits for a PR that touches the whole file. Whatever's preferred 🙂

Reply:
Sure, lets change the var name there too. I renamed the `$response` var in `wp_dashboard_setup()` because it was reused, a big no-no. Hence the `$check_browser` and `$check_php`.